File: C++Solutions/filip.cpp
#include<iostream>
using namespace std;

// function prototype
int reverse(int number);
// MAIN
int main(int argc, char const *argv[]) {

  int first_num, second_num;
  cin >> first_num >> second_num;

  int num_first, num_second;
  num_first = reverse(first_num);
  num_second = reverse(second_num);

  int max;
  (num_first > num_second) ? max = num_first : max = num_second;
  cout << max << endl;

  return 0;
}


int reverse(int number){
  int reverse = 0;
  int rem = 0;

  while (number != 0) {
    rem = number % 10;
    reverse = (reverse*10 + rem);
    number /= 10;
  }
  return reverse;
}

File: filip.py
def main():
    first_num, second_num = map(str, input().split())
    num_first = ""
    num_second = ""
    for x in range(len(first_num)):
        num_first +=(first_num[len(first_num) - (x+1)])
        num_second +=(second_num[len(second_num) - (x+1)])
    num_first = int(num_first)
    num_second = int(num_second)
    if num_first > num_second:
        print(num_first)
    else:
        print(num_second)


if __name__ == '__main__':
    main()
